apps/meteor/app/api/server/lib/emailInbox.ts

```diff
@@ -62,12 +59,12 @@ export const insertOneEmailInbox = async (
 };
 
 export const updateEmailInbox = async (
-	userId: string,
+	_userId: string,
 	emailInboxParams: Pick<IEmailInbox, '_id' | 'active' | 'name' | 'email' | 'description' | 'senderInfo' | 'department' | 'smtp' | 'imap'>,
 ): Promise<InsertOneResult<WithId<IEmailInbox>> | UpdateResult> => {
 	const { _id, active, name, email, description, senderInfo, department, smtp, imap } = emailInboxParams;
 
-	const emailInbox = await findOneEmailInbox({ userId, _id });
+	const emailInbox = await findOneEmailInbox({ _id });
 
 	if (!emailInbox) {
 		throw new Error('error-invalid-email-inbox');
```

Comment:
How about in situations like these where all we're doing is checking if an associated object exists or not, then just updating the record, we just directly call `updateOne/Many` and check `matchedCount` to validate existence?

Essentially the rule would be if both the find and update queries are using the same filter with zero dependent steps in between, just use the update query directly.

Reply:
That depends on the convention. Currently, I don't think we have a convention of either returning a 404 (the standard) or just going with the action. Some endpoints go with the action while others fail. So I'll leave them as they are 👀